### ISSUE TITLE
Optimize site for screen readers

### ISSUE THREAD

**Issue by taylortrimble** [teslaworksumn/teslaworks.net#109]
Alt tags on images tend to be missing, and in general I bet there are screen reader best practices we aren't following

**Comment by taylortrimble** [comment]
Dude,

```
<body class="bs-docs-home">
    <a class="sr-only" href="#content">Skip navigation</a>
```

That was on my mind! Awesome!

**Comment by taylortrimble** [comment]
http://accessibility.umn.edu/web-content.html
